components/Dashboard/TransactionTable.vue

```diff
@@ -1,118 +1,85 @@
-          placeholder="Type to search"/>
-      </template>
+      <el-table-column align="right">
+        <template slot="header">
+          <el-input size="mini" placeholder="Type to search"/>
+        </template>
 
-      <template >
-        <el-button
-          size="mini"
-            type="primary"
-           icon="el-icon-edit" >Edit</el-button>
-        <el-button
-          size="mini"
-          type="danger"
-         @click="dialogVisible = true" icon="el-icon-delete">Delete</el-button>
-      </template>
-    </el-table-column>
-  </el-table>
-
-
-  <el-dialog
-  title="Delete Operation"
-  :visible.sync="dialogVisible"
-  width="30%"
-  >
-  <span><b>Are you sure that you want to delete this record?</b></span>
-  <span slot="footer" class="dialog-footer">
-    <el-button @click="dialogVisible = false">Cancel</el-button>
-    <el-button type="success" @click="dialogVisible = false">Confirm</el-button>
-  </span>
-</el-dialog>
-
-</div>
+        <template slot-scope="scope">
+          <el-button size="mini" type="primary" icon="el-icon-edit">Edit</el-button>
+          <el-button
+            size="mini"
+            type="danger"
+            @click.native.prevent="deleteRow(scope.$index,facts)"
+            icon="el-icon-delete"
+          >Delete</el-button>
+        </template>
+      </el-table-column>
+    </el-table>
 
+    <el-dialog title="Delete Operation" :visible.sync="dialogVisible" width="30%">
+      <span>
+        <b>Are you sure that you want to delete this record?</b>
+      </span>
+      <span slot="footer" class="dialog-footer">
+        <el-button @click="dialogVisible = false">Cancel</el-button>
+        <el-button type="success" @click="dialogVisible = false">Confirm</el-button>
+      </span>
+    </el-dialog>
+  </div>
 </template>
 
 <script>
-  export default {
-    data() {
-       
+import axios from "axios";
+export default {
+  data() {
+    return {
+      loading: true,
+      items: [],
+      facts: [],
+      search: "",
+      dialogVisible: false
+      //   tableData: Array().fill(facts),
+      //     search: '',
+      //     dialogVisible: false
+    };
+  },
 
-      const item = {
-       
-        los: 'IFS',
-        sublos: 'IT',
-        ssublos: 'Support',
-        staffname: 'Yilmaz SEN',
-        title: 'IT Specialist',
-        location:'Beşiktaş',
-         date: '2019-02-16'
-      };
-      return {
-        tableData: Array(20).fill(item),
-          search: '',
-          dialogVisible: false
-      }
- 
-    },
-    methods: {
-      handleClose(done) {
-        this.$confirm('Are you sure to close this dialog?')
-          .then(_ => {
-            done();
-          })
-          .catch(_ => {});
-      }
+  async created() {
+    await axios
+      .get("https://pwcdemo-1c4d3.firebaseio.com/users.json")
+      .then(response => {
+        (this.loading = false), (this.facts = Object.values(response.data));
+      });
+    console.log(Object.values(this.facts));
```

Comment:
it's probably a better idea to remove `consol.log` before pushing data to github

Reply:
This line has been deleted. Thank you for your suggestion.